pkg/lib/database/filter.go

```diff
@@ -16,8 +18,10 @@ var (
 	ErrValueMustBeListSupporting = errors.New("value must be list supporting")
 )
 
-func MakeAndFilter(protoFilters []*model.BlockContentDataviewFilter, store OptionsGetter) (AndFilters, error) {
-
+func MakeAndFilter(protoFilters []*model.BlockContentDataviewFilter, store ObjectStore) (AndFilters, error) {
```

Comment:
This name of the function is strange to me :(

Reply:
the same issue, but do you have any idea how to make it batter?
possibly MakeFilterAnd can sound slightly better

Reply:
Yes, sounds better. Probably BuildFilterAnd

Reply:
Thank you, I renamed AndFilter and OrFilter to `FiltersAnd` and `FiltersOr`, think it's less confusing. Also added prefix `Filter` for all filters

Reply:
Now it's `MakeFiltersAnd`